cmd/linkchecker/linkchecker.go

```diff
@@ -37,6 +39,7 @@ var (
 
 func main() {
 	flag.StringVar(&host, "host", "", "Hostname and port of site to check.")
+	flag.StringVar(&tlsVerify, "tlsVerify", "true", "If you want to try site with invalid certificate, default: true")
```

Comment:
Did you consider using `flag.BoolVar` https://golang.org/pkg/flag/#BoolVar?

Then the usage might be something like this.

```
./linkchecker -host=www.ardanstudios.com -skiptls
```

```
flag.StringVar(&skipTLS, "skiptls", false, "To try site with invalid certificate, default: false")

...

http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: skipTLS}
```

Reply:
Thanks for the suggestion, changed it.